packages/grafana-schema/src/common/common.gen.ts

```diff
@@ -859,6 +859,10 @@ export enum VariableFormatID {
 }
 
 export interface DataSourceRef {
+  /**
+   * The plugin version that wrote the query
+   */
+  pluginVersion?: string;
```

Comment:
😬  -- i'm nervous about using this for anything real.  We have apiVersion that indicates the contract.  Including the pluginversion (that will change all the time) makes any schema difficult (eg, panel plugins!)

Reply:
hey, please check the [design doc for this](https://docs.google.com/document/d/1ph43ARg6tP18Dgh6MrngFq0vWoNk49GvR6qNsbuJLTo). There is more info about this there so let us know your opinion there 🙏